cmd/datamon/cmd/self_upgrade.go

```diff
@@ -107,7 +107,7 @@ func doCheckVersion() error {
 		return errors.New(fmt.Sprintf("could not fetch release from github repo (%s)", githubRepo)).Wrap(err)
 	}
 	if !found {
-		return errors.New(fmt.Sprintf("no matching release from github repo (%s)", githubRepo)).Wrap(err)
+		return errors.New(fmt.Sprintf("no matching release from github repo (%s)", githubRepo))
```

Comment:
wait, now we're _not_ wrapping cause errors in the errors they cause?

i thought that entire discussion about the error programming pattern change in 1.13 was to use implement and use a `getCause()`-like api (in java land .. many languages implement the error chaining motif)

Reply:
This very line has actually not err to wrap henve the remival